### DISCUSSION TITLE
[SSH Issue] discuss.gogs.io is all over google, but none of the links work?

### DISCUSSION THREAD

**Discussion by NotLazy** [gogs/gogs#6861] [Q&A] [answered]
I'm looking for docs or guides to get built in SSH working on gogs, but only discuss.gogs.io comes up in google, but the hostname doesn't resolve?

I tried using archive.org, but the page I'm trying to view hasn't been crawled since 2016, and doesn't even load, and github discussions has nothing on SSH, and the official docs have no mention of SSH.

What do I do??

I got SSH running on gogs, but I can't seem to connect.
```
gogs@gogs.example.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

What do I do?

**Comment by NotLazy** [comment, marked as answer]
I just needed to install an SSH key 🤦

**Comment by unknwon** [comment]
In case you encounter it again, you can use Wayback Machine to view old links, see  https://github.com/gogs/gogs/discussions/6863
